docs/data/routes/docs/fundamentals/services/dictionary/overview/en.md

```diff
@@ -0,0 +1,26 @@
+---
+name: overview
+routeTemplate: ./data/component-templates/article.yml
+title: Dictionary Services and APIs
+---
+
+# Dictionary Services and APIs
+
+Often multilingual apps will need a dictionary of static phrases that require translation. Commonly this would be items such as form labels, global navigation items, footers, etc. 
+
+## Sitecore endpoints for dictionary data
+
+You have several options for fetching dictionaries from Sitecore: 
+
+- [Sitecore Dictionary Service](/docs/fundamentals/services/dictionary/sitecore-dictionary-service),
+- [Sitecore Experience Edge for XM](#sitecore-experience-edge-for-xm) or 
+- [Sitecore GraphQL](#sitecore-graphql).
```

Comment:
The content in these 3 bullets can be improved, but I'm struggling to think of a suggestion. Here is the raw info:

The 3 options are
1. Call the Dictionary Service REST endpoint that's provided by Headless Services
2. Query the GraphQL endpoint that's provided by Headless Services
3. Query the GraphQL endpoint that's provided by Experience Edge for XM

In the way it's currently worded, it feels inconsistent because "Sitecore Dictionary Service" is the name of an endpoint, "Sitecore Experience Edge for XM" is the name of a product, and "Sitecore GraphQL" is a type of endpoint.

Reply:
Great point!